lib/developer_portal/lib/liquid/drops/api_spec.rb

```diff
@@ -15,6 +15,12 @@ def url
       def system_name
         @spec.system_name
       end
+
+      desc 'Returns the service of the spec if it has any or nil otherwise.'
+      def service
+        return nil unless (service = @spec.service)
```

Comment:
I didn't know if in this case I should return `nil` or an empty service drop, but I saw that for other cases we are currently returning `nil` so I kept it the same way. For example in:
https://github.com/3scale/porta/blob/029cba3e78d806c5b092ebf39f75ef3a2b081b8c/lib/developer_portal/lib/liquid/drops/service_contract.rb#L29-L33

Reply:
If it's `nil` then it can be less verbose
```ruby
return unless service = @spec.service
```